README.md

```diff
@@ -1,135 +1,131 @@
-      "**/.*",
-      "**/node_modules/**"
-    ],
-    "frameworksBackend": {
-      "region": "us-central1",
-      "minInstances": 1,
-      "maxInstances": 10
-    }
-  }
+| Field  | Type | Description |
+| ---------- | ------- | - |
+| runCommand | string |Command to start the server (e.g. "node dist/index.js"). Assume this command is run from the root dir of the workspace.This should be the productionized version of the server start command. |
+| environmentVariables| EnvVarConfig[] | Environment variables present in the server execution environment.|
+| concurrency | number | The maximum number of concurrent requests that each server instance can receive.|
+| cpu | number |The number of CPUs used in a single server instance.. |
+| memoryMiB | number | The amount of memory available for a server instance.|
+| minInstance | number |The limit on the minimum number of function instances that may coexist at a given time. |
+| MaxInstance | number | The limit on the maximum number of function instances that may coexist at a given time.|
+| EnvVarConfig.variable | string |Name of the environment variable |
+| EnvVarConfig.value | string |Value associated with the environment variable |
+| EnvVarConfig.availability | RUNTIME | Where the variable will be available, for now this will always be RUNTIME |
+
+Many of these fields are shared with apphosting.yaml as well so see [our documentation](https://firebase.google.com/docs/reference/apphosting/rest/v1beta/projects.locations.backends.builds#runconfig) for additional context.
+
+### Metadata
+
+```typescript
+interface Metadata {
+  adapterPackageName: string;
+  adapterVersion: string;
+  framework: string;
+  frameworkVersion?: string;
 }
+
 ```
 
-# Contributors
+| Field  | Type | Description |
+| ---------- | ------- | - |
+| adapterPackageName | string |Name of the adapter (this should be the npm package name) |
+| adapterVersion| string | Version of the adapter|
+| framework | string | Name of the framework that is being supported|
+| frameworkVersion | string |Version of the framework that is being supported |
 
-We'd love to accept your patches and contributions to this project. There are just a few small guidelines you need to
-follow. [See CONTRIBUTING](./CONTRIBUTING.md).
+Putting this all together a sample bundle.yaml would look as follows:
 
-## Building
+```
+> cat .apphosting/bundle.yaml
+
+version: v1
+runConfig:
+  runCommand:
+    - node
+    - dist/index.js
+  environmentVariables:
+    - variable: VAR
+      value: 8080
+      availability: RUNTIME
+  concurrency: 80
+  cpu: 2
+  memoryMiB: 512
+  minInstances: 0
+  maxInstances: 14
+    
+metadata:
+  adapterNpmPackageName: npm-name
+  adapterVersion: 12.0.0
+  frameworkNpmPackageName: framework-name
+  adapterVersion: 1.0.0
 
-```bash
-$ cd <YOUR-GIT-CHECKOUT>
-$ npm i
-$ npm run build
 ```
+
+As long as you have the bundle.yaml in this format, App Hosting will be able to deploy any server side rendered framework.
```

Comment:
```suggestion
As long as you have the bundle.yaml in this format, App Hosting will be able to deploy any framework that supports support renders.
```

Reply:
I think Daniel may have meant "that supports server side rendering."